app/views/shared/review_body.erb

```diff
@@ -27,38 +28,8 @@ Please avoid lengthy details of difficulties in the review thread. Instead, plea
 
-### Conflict of interest
-
-- [ ] As the reviewer I confirm that I have read the [JOSS conflict of interest policy](https://github.com/openjournals/joss/blob/master/COI.md) and that there are no conflicts of interest for me to review this work.
-
-### Code of Conduct
-
-- [ ] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).
-
-### General checks
-
-- [ ] **Repository:** Is the source code for this software available at the <%= link_to "repository url", paper.repository_url, :target => "_blank" %>?
-- [ ] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
-- [ ] **Version:** Does the release version given match the GitHub release (<%= paper.software_version %>)?
-- [ ] **Authorship:** Has the submitting author (<%= paper.submitting_author.github_username %>) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
-
-### Functionality
-
-- [ ] **Installation:** Does installation proceed as outlined in the documentation?
-- [ ] **Functionality:** Have the functional claims of the software been confirmed?
-- [ ] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)
-
-### Documentation
-
-- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
-- [ ] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
-- [ ] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
-- [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
-- [ ] **Automated tests:** Are there automated tests or manual steps described so that the function of the software can be verified?
-- [ ] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support
-
-### Software paper
-
-- [ ] **Authors:** Does the `paper.md` file include a list of authors with their affiliations?
-- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
-- [ ] **References:** Do all archival references that should have a DOI list one (e.g., papers, datasets, software)?
+<%- if paper.kind.present? %>
+  <%= render partial: "content/github/#{paper.kind}_review_checklist", locals: local_assigns %>
+<%- else %>
+  <%= render partial: "content/github/review_checklist", locals: local_assigns %>
```

Comment:
Does `local_assigns` make the `paper` variable available to the partial at `content/github/review_checklist` @parrish?

Reply:
Yep, it's basically a passthrough for the locals that used to render shared/review_body